### PR TITLE
Add a section about papa

### PR DESCRIPTION
**Pull request by TH3-S4LM0N**
Add a section about papa, emeralds mm.
Also written in Rust pls merge gecko

### CODE REVIEW

**Comment by TH3-S4LM0N** [comment]
100th pr

File: docs/installing-northstar/northstar-installers.md [hunk 1]
```diff
@@ -67,3 +67,9 @@ Easy to use and extensive Northstar installer and mod-manager. Supports installi
 Download and source code:
 
 {% embed url="https://github.com/BigSpice/VTOL" %}
+
+## **AnActualEmerald:** Papa
+
+cli mod manager and Northstar installer written in Rust. Available for Linux as .deb and .msi windows installer. Or build from source. Can install, uninstall, and update mods from Thunderstore
```
Comment (GeckoEidechse) [inline docs/installing-northstar/northstar-installers.md]:
```suggestion
Command-line only mod manager and Northstar installer written in Rust. Available for Linux as `.deb` and `.msi` Windows installer. Or build from source. Can install, uninstall, and update mods from Thunderstore.
```

**Comment by GeckoEidechse** [comment]
I agree we should papa to the list. My only concern is that the list is starting to get too big/confusing for someone completely new to Northstar.

Like if someone just finds out about Northstar it should be as simple as:

1. Go to wiki/website
2. Click download on installer
3. Run installer
4. Play

And with the way wiki is structured right now, we're kinda a bit far from that. For example, someone with no CLI experience and new to modding definitely shouldn't start out with papa. But if you're a server admin running Northstar docker, papa is 100% the tool you wanna use.

**Comment by TH3-S4LM0N** [comment]
Maybe make like a `extra-tools.md` that will have a overview of docker and papa and that type of stuff

**Comment by x3Karma** [comment]
> I agree we should papa to the list. My only concern is that the list is starting to get too big/confusing for someone completely new to Northstar.

I think people who read the wiki are smart enough to select one themselves.

Jokes aside, usually people when faced with multiple choices would usually try the first one in the list.

**Comment by GeckoEidechse** [comment]
> Jokes aside, usually people when faced with multiple choices would usually try the first one in the list.

Maybe? Except our first choice is kinda meh ^^

Anyway, I'm fine with merging this now and then maybe restructuring wiki later. Basically, I wanna get the flow for new players to:

```
Main page -> install -> get the right tool suggested -> download, run, done
```

**Pull request merged**